### DISCUSSION TITLE
Info: How to present the form to change the password when the user clicks the magic link sent to his email?  Does the lib include this configuration?

### DISCUSSION THREAD

**Discussion by luciocodeigniter** [codeigniter4/shield#775] [Q&A]
### PHP Version

8.2.7

### CodeIgniter4 Version

4.3.6

### Shield Version

beta

### Which operating systems have you tested for this bug?

Windows

### Which server did you use?

apache

### Database

5.7.33

### Did you customize Shield?

no

### What happened?

How to present the form to change the password when the user clicks the magic link sent to his email?

Does the lib include this configuration?

### Steps to Reproduce

How to present the form to change the password when the user clicks the magic link sent to his email?
Does the lib include this configuration?

### Expected Output

How to present the form to change the password when the user clicks the magic link sent to his email?
Does the lib include this configuration?

**Comment by datamweb** [comment]
> Does the lib include this configuration?

Yes. 
See :
https://github.com/codeigniter4/shield/blob/develop/docs/customization.md#customize-login-redirect

https://github.com/codeigniter4/shield/blob/develop/docs/quickstart.md#session-notification
Or 
https://github.com/codeigniter4/shield/blob/develop/docs/quickstart.md#event

**Comment by luciocodeigniter** [comment]
Thanks for the quick response. I will apply as suggested.
